website/docs/m-agent/usage.md

```diff
@@ -0,0 +1,35 @@
+---
+id: usage
+title: m-agent Usage
+sidebar_label: Usage
+---
+
+Upon installing m-agent, you can use it to generate a token for your Chaos Experiment. It step will require you to specify an expiry duration for your token. Tokens are valid through a minimum duration of 1 minute to a maximum of 30 days. The token can be generated in two modes:
```

Comment:
```suggestion
Upon installing m-agent, you can use it to generate a token for your Chaos Experiment. It will require you to specify an expiry duration for your token. Tokens are valid from a minimum duration of 1 minute to a maximum of 30 days. The token can be generated in two modes:
```